style.css

```diff
@@ -0,0 +1,262 @@
+/* Set every element's box-sizing to border-box */
+* {
+	box-sizing: border-box;
+}
+
+html, body {
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 18px;
+}
+
+h1, h2, h3, h4, h5 {
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container {
+	/* seems thinner via perfectpix */
+	max-width: 1024px;
+	margin: 0 auto;
+}
+
+nav ul {
+	text-align: center
+}
+
+nav li {
+	padding: 20px;
+	border-bottom: 1px solid #000;
+}
+
+nav a {
+	color: #9e9e9e;
+	text-decoration: none;
+	font-weight: bold;
+}
+
+.jumbo-text h1 {
+	font-size: 80px;
+}
+
+.jumbo {
+	padding-top: 50px;
+	margin-bottom: 80px;
+}
+
+.jumbo span {
+	display: block;
+}
+
+.jumbo-text {
+	text-align: center;
+}
+
+.jumbo-img {
+	display: none;
+}
+
+.btn {
+	display: inline-block;
+	border: 1px solid #000;
+	padding: 10px 40px;
+	text-decoration: none;
+	color: #000;
+}
+
+.animate {
+	transition: transform .7s;
+}
+
+.animate:hover {
+	transform: scale(1.1);
+}
+
+.main {
+	padding: 15px;
+}
+
+.main section {
+	margin: 20px 0;
+}
+
+.main section p {
+	margin-bottom: 40px;
+}
+
+.main img {
+	width: 100%;
+	height: 100%;
+	/* padding: 20px; */
+}
+
+.contact {
+	padding: 15px;
+}
+
+address span {
+	display: block;
+	margin: 20px 0;
+}
+
+footer {
+	text-align: center;
+	margin-top: 30px;
+}
+
+/* service page. */
+
+.services-header-img {
+	width: 100%;
+	padding-top: 60px;
+}
+
+.services-intro h1 {
+	font-size: 30px;
+	margin-top: 30px;
+}
+
+.services-intro {
+	/* margin-top: 20px; */
+	padding-bottom: 20px;
+	border-bottom: 1px solid #000;
+}
+
+.service-list {
+	margin-top: 30px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	border-bottom: 1.5px solid #000;
+}
+
+.service-item {
+	background-color: #f4f3f4;
+	width: 100%;
+	border: 1px solid #000;
+	padding: 10px;
+	margin-bottom: 25px;
+}
+
+.service-item p {
+	margin-bottom: 10px;
+}
+
+.learn {
+	border-radius: 5px;
+	background-color: #fff;
+}
+
+.facts {
+	margin: 30px 0 50px 0;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+
+.fact-item ul {
+	list-style-type: square;
+	margin-left: 16px;
+}
+
+.fact-item {
+	width: 100%;
+}
+
+.fact-item p {
+	margin-bottom: 20px;
+}
+
+.fact-item ul li {
+  margin-bottom: 15px;
+}
+
+/* tablet */
+@media (min-width: 500px) {
+	.container {
```

Comment:
you may want some margin on the mobile version of the services page, if you check the edges of the content are so close to the width that it almost looks cut off